src/lib/index.ts

```diff
@@ -34,9 +28,9 @@ export default async function generateCoverageReport(
   const data = await getCoverage({
     strict: options.strict,
     debug: options.debug,
-    ignoreFiles: options["ignore-files"],
-    ignoreCatch: options["ignore-catch"],
-    ignoreUnread: options["ignore-unread"],
+    ignoreFiles: options.ignoreFiles,
+    ignoreCatch: options.ignoreCatch,
+    ignoreUnread: options.ignoreUnread,
     cache: options.cache
```

Comment:
At this point might as well pass option to `getCoverage`

Reply:
Not a show blocker anyway

Reply:
True, good future improvement